### PR TITLE
interfaces: network-manager: add AppArmor rule for configuring bridges

### PR DESCRIPTION
**Pull request by IsaacJT**
The network-manager interface does not currently provide the AppArmor rules for configuring bridges, resulting in the following denial (among others):

```
= AppArmor =
Time: Apr 27 10:57:12
Log: apparmor="DENIED" operation="open" profile="snap.network-manager.networkmanager" name="/sys/devices/virtual/net/br0/bridge/vlan_filtering" pid=2863 comm="NetworkManager" requested_mask="w" denied_mask="w" fsuid=0 ouid=0
File: /sys/devices/virtual/net/br0/bridge/vlan_filtering (write)
Suggestions:
* adjust program to not access '/sys/devices/virtual/net/br0/bridge/vlan_filtering'
* adjust program to not access '/sys/devices/virtual/net/br[0-9]*/bridge/vlan_filtering'
```

This can be reproduced by creating a bridge interface using `nmcli` from the `network-manager` snap:

```
$ sudo nmcli con add type bridge ifname br0 bridge.vlan-filtering yes
Connection 'bridge-br0' (3981b337-92a9-45a7-b4a2-8478741aa876) successfully added.
$ sudo con up bridge-br0
Error: Connection activation failed: The device could not be readied for configuration
```

This PR adds the necessary AppArmor rule for configuring bridges through the sysfs interface. The AppArmor rule was copied from:

https://github.com/snapcore/snapd/blob/f960ee96f2d4bae99ef25e63764540ffbeacc44d/interfaces/builtin/network_control.go#L218.

### CODE REVIEW

**Review by bboozzoo — APPROVED** [review]
Thank you. Given that network-control plug allows that, it makes sense to allow the same for the network-manager slot.

**Review by mardy — APPROVED** [review]
LGTM, thanks! But let's wait for somebody from the security team to review it.

**Review by alexmurray — APPROVED** [review]
LGTM!

**Pull request merged**
